lib/Gitlab/Api/AbstractApi.php

```diff
@@ -190,6 +191,29 @@ protected function encodePath($path)
         return str_replace('.', '%2E', $path);
     }
 
+    /**
+     * Create a new OptionsResolver with page, per_page and sort options.
+     *
+     * @return OptionsResolver
+     */
+    protected function createOptionsResolver()
```

Comment:
why is this its own method? its only used once... or is this a first step and more methods will be refactored to this?